ClientCredentialsKeypairs/Fhi.ClientCredentialsKeypairs.csproj

```
@@ -5,7 +5,7 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <LangVersion>latest</LangVersion>
-    <Version>1.2.0</Version>
+    <Version>1.2.1</Version>
```

This is a new feature, not a bug fix, so...  1.3.0
```suggestion
    <Version>1.3.0</Version>
```